en/components/banner.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Banner - Design System Component
+_description: The Banner Component is used to show an interactive information message or notification in a prominent way. 
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+# Banner
+
+Use the Banner Component to display a prominent message and optionally actions related to it. The Banner is visually identical to the [Ignite UI for Angular Banner Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/banner)
+
+## Banner Demo
+
+<img class="responsive-img" src="../images/banner_demo.png" srcset="../images/banner_demo@2x.png 2x" />
+
+## Message
+
+The Banner...
+
+## Actions
+
+The Banner...
```

Comment:
The Actions consist of two Button components. If you need you can turn one or both off by setting them to ~No Symbol for Sketch or turning off the layer's visibility in Adobe XD.

Reply:
![Actions@2x](https://user-images.githubusercontent.com/57216334/128358437-9cf0b63d-45f4-48b2-b57f-aec43537ccf2.png)
![Actions](https://user-images.githubusercontent.com/57216334/128358443-dab2e86c-afcf-4177-b087-fa35f18601f4.png)

Reply:
```suggestion
The Actions consist of two Button components. If you need you can turn one or both off by setting them to ~No Symbol for Sketch or turning off the layer's visibility in Adobe XD.
```